README.md

```diff
@@ -96,32 +96,37 @@ epcisDocument.addEvent(event).addEvent(event2);
 console.log(epcisDocument.toString());
 ```
 
-This example would output:
+This example would output (as a string):
 
 ```json
 {
-  "type": "EPCISDocument",
-  "@context": "https://gs1.github.io/EPCIS/epcis-context.jsonld",
-  "schemaVersion": "2",
+  "isA":"EPCISDocument",
+  "@context":"https://gs1.github.io/EPCIS/epcis-context.jsonld",
+  "schemaVersion":"2",
+  "creationDate":"2022-03-15T09:48:06.047Z",
   "epcisBody": {
-    "eventList": [
+    "eventList": [ 
       {
-        "type": "ObjectEvent",
-        "action": "OBSERVE",
-        "eventTime": "2005-04-03T20:33:31.116-06:00",
-        "eventTimeZoneOffset": "-06:00"
+        "eventTimeZoneOffset":"-06:00",
+        "eventTime":"2005-04-03T20:33:31.116-06:00",
+        "isA":"ObjectEvent",
+        "action":"OBSERVE" 
       },
       {
-        "type": "ObjectEvent",
-        "action": "OBSERVE",
-        "eventTime": "2005-04-03T21:33:31.116-06:00",
-        "eventTimeZoneOffset": "-06:00"
+        "eventTimeZoneOffset":"-06:00",
+        "eventTime":"2005-04-03T21:33:31.116-06:00",
+        "isA":"ObjectEvent",
+        "action":"OBSERVE"
       }
     ]
   }
 }
 ```
 
+The latest version of the SDK enables you to easily search among all attributes of each Common Business Vocabulary.
+In order to do this, for example, you can import `actionType` as we did in the script above. Then, by typing `actionType.` as a parameter of the `setAction` method of the `ObjectEvent` class, you will be displayed all the attributes associated to this particular CBV.
```

Comment:
I would put this in a dedicated section, and not in 'Adding events to an EPCIS 2.0 document'